src/app/retrospective/sprint-detail/sprint-detail.component.html

```diff
@@ -0,0 +1,52 @@
+<mat-card>
+  <div class="flex-container">
+    <div>
+      <mat-card-title>{{sprintDetails.Title}}</mat-card-title>
+    </div>
+   <div>
+     <span matTooltip="Retrospective dashboard" matTooltipPosition="before">
+       <span (click)="navigateToRetrospectiveDashboard()" class="circular-icon">
+         &nbsp;<mat-icon class="back-icon">arrow_back</mat-icon>&nbsp;
+       </span>
+     </span>
+   </div>
+  </div>
+  <mat-card-content class="flex-container">
```

Comment:
use proper class names instead of `flex-container`, `flex-element`, `circular-icon`

Reply:
Added proper scoping in scss for better understandability

Reply:
Added proper scoping in scss for better understandability